python/streamlit.qmd

```diff
@@ -1,3 +1,35 @@
 # Streamlit {.unnumbered}
```

Comment:
assuming this is kinda a intermediary from you standing this up and not for review as you'll expand it with the python section work?

Reply:
Correct.